heft-plugins/heft-webpack5-plugin/src/WebpackConfigurationLoader.ts

```diff
@@ -92,3 +98,34 @@ export class WebpackConfigurationLoader {
     }
   }
 }
+
+async function findWebpackConfigAsync(buildFolder: string): Promise<IWebpackConfigFileNames> {
+  try {
+    const folderItems: FolderItem[] = await FileSystem.readFolderItemsAsync(buildFolder);
+    const dev: string[] = [];
+    const prod: string[] = [];
+
+    for (const folderItem of folderItems) {
+      if (folderItem.isFile()) {
+        if (folderItem.name.match(/^webpack.dev.config\.(cjs|js|mjs)$/)) {
+          dev.push(folderItem.name);
+        } else if (folderItem.isFile() && folderItem.name.match(/^webpack.config\.(cjs|js|mjs)$/)) {
```

Comment:
```suggestion
        } else if (folderItem.name.match(/^webpack.config\.(cjs|js|mjs)$/)) {
```

Reply:
AH dangit - I missed it. @iclanton mind if we commit here?